definitions/npm/react-redux_v5.x.x/flow_v0.63.0-/react-redux_v5.x.x.js

```diff
@@ -57,7 +67,7 @@ declare module "react-redux" {
     S: Object,
     SP: Object,
     RSP: Object,
-    CP: $Diff<OmitDispatch<ElementConfig<Com>>, RSP>,
+    CP: $Diff<ElementConfig<Com>, RSP>,
```

Comment:
Why OmitDispatch was removed? Usually react-redux passes it to component so it's not required to pass it explicitly.

Reply:
@LoganBarnett ping :)

Reply:
@LoganBarnett I tried the version in our codebase and look like it relies that connect passes the dispatch to component in this case.

Reply:
@LoganBarnett I guess correct type in omit for dispatch is actually `{dispatch?: Dispatch<*>}`, so this might fix the problem: `declare type OmitDispatch<Component> = $Diff<Component, {dispatch?: Dispatch<*>}>;`

Reply:
@villesau got it - okay I've made that change. How is your codebase encountering this issue? I'd like to capture it in a test if possible.

Also, if you know of ways to capture type coverage itself as a test I'm all ears. So far it feels like I'm spitballing just to get these to add coverage and any changes in the future could just remove them without failing anything.